src/com/facebook/buck/jvm/java/intellij/ij-module.st

```diff
@@ -49,7 +49,14 @@
         %if(generatedFolder.isAndroidResources)% type="java-resource" %endif%/>
     </content>
 }%
+
+%if(!androidFacet.enabled && java_sdk)%
+    <orderEntry type="jdk" jdkName="%java_sdk%" jdkType="JavaSDK" />
```

Comment:
> This is purely scoped to java_sdk. If someone wants to use groovysdk, they would need to be a separate config key groovy_sdk in the intellij block of buckconfig and it could be used here.

I think one of my comments wasn't clear. I didn't mean you should add support for more kind of language SDKs, I meant that you have no guarantee that these SDKs are defined in IntelliJ's config nor that they are correctly set up. To put it another way - you're effectively hardcoding strings and expecting those to be correctly set up on every machine you run `buck project` on.

> The auto detection change should not be part of this change imo
Agreed, that was not what I was asking for. My question was "do you want the code?", not "merge it into this PR". The real question behind it is: are you planning more changes to this feature (project generation)?